### ISSUE TITLE
Can Antidote call contact over Tor?

### ISSUE THREAD

**Issue by bucky0970** [Antidote-for-Tox/Antidote#292]
I already run up a Tor proxy in app, how to use the proxy make a call?

**Comment by dvor** [comment]
I'm not sure how Tox proxy works on iOS, I need to investigate that.

Currently in Antidote there is no configuration for proxy settings.

**Comment by dvor** [comment]
@bucky0970 can you please share a link to Tor proxy app so I can check and test it?

**Comment by bucky0970** [comment]
Thanks for your reply~
I use [CPAProxy](https://github.com/ursachec/CPAProxy) , after proxy start up, you can get a socks5 proxy ip and port. And I found in (Project Navigator)Antidote/Pods/Pods/toxcore/tox_api.m line:67
```
void tox_options_default(struct Tox_Options *options)
{
    if (options) {
        struct Tox_Options default_options = { 0 };
        *options = default_options;
        tox_options_set_ipv6_enabled(options, true);
        tox_options_set_udp_enabled(options, true);
        tox_options_set_proxy_type(options, TOX_PROXY_TYPE_NONE );
        tox_options_set_hole_punching_enabled(options, true);
        tox_options_set_local_discovery_enabled(options, true);
    }
}
```
It seems to be able set proxy, It has 3 options. TOX_PROXY_TYPE_NONE, TOX_PROXY_TYPE_SOCKS5 and TOX_PROXY_TYPE_HTTP, but I have no idea where to config proxy ip and port.
I saw Tox wiki, in command line, Tox can communicate over Tor.([wiki link](https://wiki.tox.chat/users/tox_over_tor_tot)).

**Comment by dvor** [comment]
Yes, it would be nice to investigate on including CPAProxy (or any other) to Antidote.

**Comment by hsn10** [comment]
For my use pattern it would be enough to add support for SOCKS 5. As I understand from previous post, there is already SOCKS 5 support, just GUI is missing for its configuration.
